contributions/api/controllers/contributions.py

```diff
@@ -459,14 +459,15 @@ def allcapabilitiessearch(token_info=None, name=None):
         out_json = mongoutils.get_result(coll_contribution, query)
     else:
         try:
-            query = query_params.format_query_capability(name, query)
+            # if there is a query string, query the contributions
```

Comment:
Since we decided to remove the query parameter for /contributions/talents and /contributions/capabilities endpoints, do we need the text search query here and in alltalentssearch method?

Reply:
updated. removed unnecessary checks

Reply:
Thanks.